include/phoenix/if_ether.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * phoenix/if_ether.h
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Ziemowit Leszczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _LIBPHOENIX_PHOENIX_IF_ETHER_H_
```

Comment:
side note (didn't catch it earlier): in `phoenix` subfolder kernel headers are being installed - I don't know if it's a good idea to mix `libphoenix` headers in the same dir. Right now it doesn't matter, but we should think of some long-term policy ;).

Not a merge blocker for now.

Reply:
Linux keeps these files in linux dir while BSD in net dir.